build.gradle.kts

```diff
@@ -26,56 +21,25 @@ sourceSets {
     }
 }
 
-// Import variables from gradle.properties file
-val pluginGroup: String by project
-// `pluginName_` variable ends with `_` because of the collision with Kotlin magic getter in the `intellij` closure.
-// Read more about the issue: https://github.com/JetBrains/intellij-platform-plugin-template/issues/29
-val pluginName_: String by project
-val pluginVersion: String by project
-val pluginSinceBuild: String by project
-val pluginUntilBuild: String by project
+group = "com.github.cross_language_cpp.djinni_intellij_plugin"
 
-val platformType: String by project
-val platformVersion: String by project
-val platformPlugins: String by project
-val platformDownloadSources: String by project
-
-group = pluginGroup
-version = pluginVersion
+version = gitVersion()
```

Comment:
This generates a version-string from `git describe`. It's kind of the same thing like in the generator.